### PR TITLE
Screpyy96 welcome done

### PR DESCRIPTION
**Pull request by screpyy96**

### CODE REVIEW

File: web/asset/demo-files/demo.css [hunk 1]
```diff
@@ -0,0 +1,152 @@
+body {
+  padding: 0;
+  margin: 0;
+  font-family: sans-serif;
+  font-size: 1em;
+  line-height: 1.5;
+  color: #555;
+  background: #fff;
+}
```
Comment (RalucaRbc) [inline web/asset/demo-files/demo.css:9]: You need a new  empty line between the divs

**Review by RalucaRbc — APPROVED** [review]

File: web/asset/demo-files/demo.css [hunk 2]
```diff
@@ -0,0 +1,152 @@
...
+h1 {
+  font-size: 1.5em;
+  font-weight: normal;
+}
+small {
+  font-size: .66666667em;
```
Comment (marianzburlea) [inline web/asset/demo-files/demo.css:15]: add a leading zero `font-size: .66666667em;` to `font-size: 0.66666667em;`

File: web/asset/demo-files/demo.css [hunk 3]
```diff
@@ -0,0 +1,152 @@
...
+}
+.fgc0 {
+  color: #000;
+}
+p {
+  margin-top: 1em;
+  margin-bottom: 1em;
+}
+.mvm {
+  margin-top: .75em;
```
Comment (marianzburlea) [inline web/asset/demo-files/demo.css:80]: add a leading zero to the values

File: web/asset/demo-files/demo.css [hunk 4]
```diff
@@ -0,0 +1,152 @@
...
+  margin-bottom: .75em;
+}
+.mtn {
+  margin-top: 0;
+}
+.mtl, .mal {
```
Comment (marianzburlea) [inline web/asset/demo-files/demo.css:86]: It's idea to separate selectors on a new line, to increase readablity:
```
.mtl,
.mal {
  margin-top: 1.5em;
}
```

**Review by marianzburlea — APPROVED** [review]

**Pull request merged**